src/Pos/Wallet/Web/Api.hs

```diff
@@ -32,46 +34,76 @@ type WalletApi =
      :> Post '[JSON] (Either WalletError ())
     :<|>
      -------------------------------------------------------------------------
+     -- Wallet sets
+     -------------------------------------------------------------------------
+     "api"
+     :> "walletSets"
+     :> Capture "walletSetId" (CAddress WS)
+     :> Get '[JSON] (Either WalletError CWalletSet)
+    :<|>
+     "api"
+     :> "walletSets"
+     :> Get '[JSON] (Either WalletError [CWalletSet])
+    :<|>
+     "api"
+     :> "walletSets"
+     :> "new"
+     :> Capture "passphrase" CPassPhrase
+     :> ReqBody '[JSON] CWalletSetInit
+     :> Post '[JSON] (Either WalletError CWalletSet)
+    :<|>
+     "api"
+     :> "walletSets"
+     :> "restore"
+     :> Capture "passphrase" CPassPhrase
+     :> ReqBody '[JSON] CWalletSetInit
+     :> Post '[JSON] (Either WalletError CWalletSet)
+    :<|>
+     "api"
+     :> "walletSets"
+     :> "keys"
+     :> ReqBody '[JSON] Text
+     :> Post '[JSON] (Either WalletError CWalletSet)
+    :<|>
+     -------------------------------------------------------------------------
      -- Wallets
      -------------------------------------------------------------------------
      "api"
      :> "wallets"
-     :> Capture "walletId" CAddress
+     :> Capture "walletId" CWalletAddress
```

Comment:
It historically happened to have such name, and I agree, better to call it `CWalletId` (and same for `CAccount`)